lib/src/firebase_image.dart

```diff
@@ -35,18 +35,19 @@ class FirebaseImage extends ImageProvider<FirebaseImage> {
   /// [scale] Default: 1.0. The scale to display the image at (optional)
   /// [maxSizeBytes] Default: 2.5MB. The maximum size in bytes to be allocated in the device's memory for the image (optional)
   /// [cacheRefreshStrategy] Default: BY_METADATA_DATE. Specifies the strategy in which to check if the cached version should be refreshed (optional)
-  /// [firebaseApp] Default: the default Firebase app. Specifies a custom Firebase app to make the request to the bucket from (optional)
+  /// [firebaseApp] Specifies a custom Firebase app to make the request to the bucket from (required)
   FirebaseImage(
     String location, {
     this.shouldCache = true,
     this.scale = 1.0,
     this.maxSizeBytes = 2500 * 1000, // 2.5MB
     this.cacheRefreshStrategy = CacheRefreshStrategy.BY_METADATA_DATE,
-    this.firebaseApp,
+    required this.firebaseApp,
```

Comment:
This breaking change should be documented in the change log and/or readme.

Reply:
Is this _breaking change_ necessary?

Reply:
Agreed. firebaseApp is no longer required.